src/CommunityPatchLauncher/Windows/MainWindow.xaml.cs

```diff
@@ -52,7 +52,8 @@ private async Task UpgradetoSettlersUnited()
         {
             await ZipInstallerAsync();
             string URI = "https://files.settlers-united.com/Settlers-United.exe";
-            DownloadFileAsync(URI, "SettlersUnitedSetup.exe", Properties.Resources.Setters_United_Beta, true);
+            var filename = System.Environment.GetEnvironmentVariable("TEMP") + "\\SettlersUnitedSetup.exe";
```

Comment:
You should use `Path.GetTempPath()` to get the temp path, it's more reliable
You should create system file paths with `Path.Combine()`